src/go/rpk/pkg/cli/cmd/cluster/partitions/partitions.go

```diff
@@ -31,6 +31,7 @@ func NewPartitionsCommand(fs afero.Fs) *cobra.Command {
 
 	cmd.AddCommand(
 		NewBalancerStatusCommand(fs),
+		NewMovementCancelCommand(fs),
```

Comment:
I think both of these commands should be internal functions, not externally exported.